Cython/Utility/ModuleSetupCode.c

```diff
@@ -1410,9 +1410,12 @@ static CYTHON_SMALL_CODE PyObject* ${pymodule_create_func_cname}(PyObject *spec,
 
 /////////////// CodeObjectCache.proto ///////////////
 
-#if !CYTHON_COMPILING_IN_LIMITED_API
 typedef struct {
+#if CYTHON_COMPILING_IN_LIMITED_API
+    PyObject* code_object;
```

Comment:
The only difference in this section is that the limited API doesn't know about `PyCodeObject`. Otherwise it's all the same.